baseplate/clients/thrift.py

```diff
@@ -71,7 +76,7 @@ class ThriftContextFactory(ContextFactory):
 
     """
 
-    PROM_PREFIX = "bp_thrift_pool"
+    PROM_PREFIX = "thrift_pool"
```

Comment:
🔕 the name here is a tad confusing since there are other labels that aren't prefixed with `_pool_` (and it's not clear that this is for labels, not metric names)